app/translations/en.json

```
@@ -1376,6 +1376,9 @@
                 "icon": {
                     "name": "Icon",
                     "description": "Icon displayed on the button"
+                },
+                "labelFilterRules": {
+                    "name": "Label filter rules"
```

I guess in the context of "Create Deployment Button" widget more accurate would be to have name indicating that the filter is for blueprints and not deployments, so I suggest to have here:
```suggestion
                    "name": "Blueprint label filter rules"
```

---

[c3c045f](https://github.com/cloudify-cosmo/cloudify-stage/pull/1935/commits/c3c045fdf0ac0f1897def6683d5db9ac92632098)